template/map.jinja

```diff
@@ -8,17 +8,17 @@
 {%- import_yaml tplroot ~ "/osfamilymap.yaml" as osfamilymap %}
 {%- import_yaml tplroot ~ "/osmap.yaml" as osmap %}
 {%- import_yaml tplroot ~ "/osfingermap.yaml" as osfingermap %}
+{%- set _lookup = salt['config.get']('template:lookup', default={}) %}
+{%- set _config = salt['config.get']('template', default={}) %}
 
-{%- set defaults = salt['grains.filter_by'](default_settings,
-    default='template',
+{%- set template = salt['grains.filter_by'](default_settings, default='template',
     merge=salt['grains.filter_by'](osfamilymap, grain='os_family',
       merge=salt['grains.filter_by'](osmap, grain='os',
         merge=salt['grains.filter_by'](osfingermap, grain='osfinger',
-          merge=salt['config.get']('template:lookup', default={})
+          merge=salt['grains.filter_by']({'lookup': _lookup}, default='lookup',
+            merge=salt['grains.filter_by']({'config': _config}, default='config')
```

Comment:
I doubt we need to call `grains.filter_by` for the last merge value. There is no "merging" here. Why not simply
```suggestion
            merge=_config
```
?